src/voxel.cpp

```diff
@@ -23,7 +23,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "nodedef.h"
 #include "util/directiontables.h"
 #include "util/timetaker.h"
-#include <cstring>  // memcpy, memset
+#include <cstring> // memcpy, memset
```

Comment:
you can also drop those comments, IDE are here for that, no need to comment headers